### PR TITLE
Fix isHidden() method and add allDisabled() method

### PR DESCRIPTION
**Pull request by annetill**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

The `isHidden()` method could means that all the controller elements are disabled. In that case, the target V equation must be deactivated.
Furthermore, there is a mistake in the method to check if a voltage control is hidden or not. The unit test does not show the bug because the defaut behvior on main branch is 'false'. I suggest a more clear version.

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

### CODE REVIEW

File: src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java [hunk 1]
```diff
@@ -224,7 +224,7 @@ static <T extends LfElement> void updateRemoteVoltageControlEquations(VoltageCon
                         .setActive(false);
                 equationSystem.getEquation(controllerElement.getNum(), ctrlEqType)
                         .orElseThrow()
-                        .setActive(!controllerElement.isDisabled());
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java:227]: Indeed all controller elements are enabled at this stage, so we can directly put 'true'.

File: src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java [hunk 2]
```diff
@@ -328,7 +328,7 @@ public static void updateGeneratorVoltageControl(GeneratorVoltageControl voltage
         checkNotDependentVoltageControl(voltageControl);
         LfBus controlledBus = voltageControl.getControlledBus();
         if (voltageControl.isLocalControl()) {
-            if (voltageControl.isHidden()) {
+            if (voltageControl.isHidden()) { // FIXME
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java:331]: Need to discuss with you about what does it means for generator voltage control which priority is 0.

File: src/main/java/com/powsybl/openloadflow/network/VoltageControl.java [hunk 3]
```diff
@@ -143,11 +143,19 @@ public boolean isHidden() {
         List<VoltageControl<?>> voltageControls = findVoltageControlsSortedByPriority(controlledBus);
         if (voltageControls.isEmpty()) {
             return true; // means all disabled
-        }
-        // we should normally have max 3 voltage controls (one of each type) because already merged
-        if (voltageControls.size() > 1) {
```
Comment (geofjamg) [inline src/main/java/com/powsybl/openloadflow/network/VoltageControl.java:148]: This code is totally equivalent to previous one. if only one voltage control in the list it is necessary "this" (because we start from it) and so on "voltageControls.get(0) != this" is false.
I would revert this change.

Comment (annetill) [reply src/main/java/com/powsybl/openloadflow/network/VoltageControl.java:148]: ´This’ (imagine generator voltage control) could have all its controllers disabled, but the controlled bus can have another voltage control (imagine with transformer) with enabled controllers. In that case, the list of voltage control is of size 1. ´This’ is hidden for me.

Comment (geofjamg) [reply src/main/java/com/powsybl/openloadflow/network/VoltageControl.java:148]: yes, agree

**Review by geofjamg — APPROVED** [review]
